packages/fancy_logger/test/src/writable_log_record.dart

```diff
@@ -0,0 +1,35 @@
+// ignore_for_file: prefer_const_constructors
```

Comment:
I'm not sure, but as I remember, files with tests must contains `_test` suffix in file name

Reply:
[Exactly](https://dartcodemetrics.dev/docs/rules/common/prefer-correct-test-file-name), thank you!